tests/test_complex_application.py

```diff
@@ -173,3 +193,67 @@ def test_complex():
 
     for path in [csv_path_sum, excel_path_sum, csv_path_out, excel_path_out]:
         os.remove(path)
+
+
+# def test_same_pipeline_submission_with_overlap_datanode():
+#     Config.configure_job_executions(mode=JobConfig._STANDALONE_MODE, nb_of_workers=1)
+#     _Scheduler._update_job_config()
+
+#     # d1 ---- t1 ---- d2 ---- t2
+#     # t3 ---- d2
+
+#     csv_path_inp = os.path.join(pathlib.Path(__file__).parent.resolve(), "data_sample/example.csv")
+
+#     dn1_inp_csv = Config.configure_csv_data_node("dn1_inp_csv", default_path=csv_path_inp)
+#     dn2_div = Config.configure_pickle_data_node("dn2_div")
+
+#     task1_div_csv_with_constant = Config.configure_task(
+#         "task1_div_csv_with_constant", div_constant_with_sleep, input=dn1_inp_csv, output=dn2_div
+#     )
+#     task2_print_pickle = Config.configure_task("task2_print_pickle", print_with_sleep, input=dn2_div)
+#     task3_populate_constant = Config.configure_task(
+#         "task3_populate_constant", return_a_number_with_sleep, output=dn2_div
+#     )
+
+#     pipeline_div_print_config = Config.configure_pipeline(
```

Comment:
????

Reply:
This is the test to test the effect of unlock_editition when submitting the same pipeline multiple times. So far, I see no problem regarding the topic. But I think the test is a bit immature to conclude that the question is solved.